Tribler/Test/Community/Market/test_community.py

```diff
@@ -338,7 +338,7 @@ def test_partial_trade(self):
 class TestMarketCommunityTwoNodes(TestMarketCommunityBase):
     __testing__ = True
 
-    @twisted_wrapper(2)
+    @inlineCallbacks
```

Comment:
How do we specify a timeout for a specific test now?

Reply:
By setting the `timeout` property of the test class instance.
See https://twistedmatrix.com/documents/16.3.1/api/twisted.trial.unittest.TestCase.html#timeout

Reply:
Ah I see, is there a reason why these timeouts are not added back in these tests?

Reply:
Yes. I was too lazy to do it ))

Reply:
In fact, we have to specify timeout as a property of the test *method* itself. Like
```python
def test_some(self):
    bla
test_some.timeout = 3 #seconds
```
I don't know if it is convenient or not. Maybe we could devise a wrapper or something like that.